### PR TITLE
Exclude selected items for entities picker

### PR DESCRIPTION
**Pull request by piitaya**
## Type of change

- [ ] Dependency upgrade
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/components/entity/ha-entities-picker.ts [hunk 1]
```diff
@@ -95,7 +96,10 @@ class HaEntitiesPickerLight extends LitElement {
               .excludeEntities=${this.excludeEntities}
               .includeDeviceClasses=${this.includeDeviceClasses}
               .includeUnitOfMeasurement=${this.includeUnitOfMeasurement}
-              .entityFilter=${this._entityFilter}
+              .entityFilter=${this._getEntityFilter(
```
[the remark this answers is not in the document]
Comment (piitaya) [reply src/components/entity/ha-entities-picker.ts:99]: Like this ?
```ts
.entityFilter=${guard(
  [this.value, this.entityFilter],
  () => this._entityFilter
)}
```

[the remark this answers is not in the document]
Comment (steverep) [reply src/components/entity/ha-entities-picker.ts:99]: Yes, but you wouldn't need the pseudo getter.  Just giving `guard` the original function would work.

[the remark this answers is not in the document]
Comment (piitaya) [reply src/components/entity/ha-entities-picker.ts:99]: i just tested and the getter is needed to re-evaluate `value` and `entityFilter` properties (even if we use guard)

**Review by bramkragten — APPROVED** [review]

**Comment by ildar170975** [comment]
Entities card.
UI editor.
Will a user be able to add same entity twice?

If not - consider this use case with sun:
- state;
- elevation.

Will it be possible?

**Comment by piitaya** [comment]
Entities card doesn't use `entities-picker` but a custom one because of duplicate entity selection (the use-case you describe), the edit button for additional config and the special row type display.

**Pull request merged**